src/garage/np/exploration_strategies/exploration_strategy.py

```diff
@@ -1,10 +1,72 @@
-class ExplorationStrategy:
+"""Exploration Strategy API used by off-policy algorithms."""
+import abc
 
-    def get_action(self, t, observation, policy, **kwargs):
+
+# This should be an ABC inheritting from garage.Policy, but that doesn't exist
+# yet.
+class ExplorationStrategy(abc.ABC):
```

Comment:
wait -- why leave this AND ExplorationPolicy around?

Reply:
Whoops. Just git things.